webpack/templates/route.hbs

```diff
@@ -0,0 +1,8 @@
+ {
+    path: '/{{ path }}',
```

Comment:
This should be `path: '{{ path }}'`, the prefixed slash is a "reset" and does not always have to be used!